tools/stress-cluster/cluster/kubernetes/stress-test-addons/templates/stress-test-static-secret-provider.tpl

```diff
@@ -0,0 +1,37 @@
+{{- define "stress-test-addons.static-secrets" -}}
+{{- /* The subscriptionConfig key allows a user to use a custom set of subscription+creds uploaded manually to the static keyvault */}}
+{{- /* This template finds all unique subscription configs in scenarios and creates a single secret provider for each */}}
+{{- $global := . }}
+{{- $subChart := get $global.Values "stress-test-addons" }}
+{{- $subConfigs := list (get $subChart.subscription $subChart.env) }}
```

Comment:
is there a particular reason why we are using different subConfigs for stress-test-static-secret-provider.tpl and _util.tpl?

Reply:
The subConfig in the util template populates the subscriptionConfig value set for an individual scenario. The `$subConfigs` here collects all subscription configs defined across a matrix config so we can create a secret provider for each.